example/metric/otlpgrpc/README.md

```diff
@@ -0,0 +1,24 @@
+# OTLP Metric with Google Auth Example
+
+Run this sample to connect to an endpoint that is protected by GCP authentication.
+
+#### Prerequisites
+
+Get Google credentials on your machine:
+
+```sh
+gcloud auth application-default login
+```
+
+#### Run the Sample
+
+```sh
+# export necessary OTEL environment variables
+export PROJECT_ID=<project-id>
+export OTEL_EXPORTER_OTLP_ENDPOINT=<endpoint>
+export OTEL_RESOURCE_ATTRIBUTES="gcp.project_id=$PROJECT_ID,service.name=otlp-sample,service.instance.id=1"
```

Comment:
Could be enhanced with resource detectors ?

Reply:
Not sure what you mean.  Other than service.instance.id, you can't detect these attributes